### PR TITLE
Add image-builder-packer image

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

File: image-builder-packer/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+1
```
Comment (rporres) [inline image-builder-packer/VERSION:1]: semver is your friend :)

**Comment by rporres** [comment]
hi @Gundersanne . Can you give us some context about this PR? We use this repo to avoid having repos with single dockerfiles scattered all over the place, although we haven't thought about the possibility of tenants using it. Not that I'm opposed, it's just this came as a surprise :-D

**Comment by Gundersanne** [comment]
@rporres Made this a bit prematurely, this container takes  a while to build and we need it for the packer build job, and someone suggested adding it here. However due to limitations it will be easier to rebuild it every job. So I'll just close, sorry for the noise!

**Comment by rporres** [comment]
no worries. We use this repo for that too, as you can by the `-base` names of some of the images :)

**Pull request closed without merging**